Cargo.toml

```diff
@@ -26,3 +26,4 @@ futures = "0.3"
 chrono = "0.4"
 indexmap = { version = "1.6", features = [ "serde", "serde-1" ] }
 merge = "0.1"
+gpgme = "0.10.0"
```

Comment:
Is the latest `0.11.0` version of `gpgme` not compatible with some other dependencies used in `update-daemon`?

Reply:
oh, nvm, it's actually the `rusc` version that is too low, not `toml`